src/Content/MemoryContentStorageHandler.php

```diff
@@ -79,6 +80,28 @@ public function read(VersionId $versionId, Language $language): array
 		return $this->cache->get($this->cacheId($versionId, $language));
 	}
 
+	/**
+	 * Searches and replaces one or multiple strings
+	 *
+	 * @throws \Kirby\Exception\NotFoundException If the version does not exist
+	 */
+	public function replaceStrings(
+		VersionId $versionId,
+		Language $language,
+		array $map
+	): void {
+		$fields = $this->read($versionId, $language);
+		$fields = A::map(
+			$fields,
+			fn ($field) => str_replace(
+				array_keys($map),
+				array_values($map),
+				$field
+			)
+		);
+		$this->update($versionId, $language, $fields);
```

Comment:
Couldn't this part be in the ContentStorageHandler class as a default instead of declaring it as abstract method?

Reply:
Good idea!